resotool/resotool/settings.py

```diff
@@ -122,4 +124,25 @@
 
 # Email
 
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_BACKEND = 'djcelery_email.backends.CeleryEmailBackend'
+
+CELERY_EMAIL_TASK_CONFIG = {
+    'ignore_result': False,
+}
+CELERY_EMAIL_CHUNK_SIZE = 1
+
+# Celery stuff
+
+CELERY_BROKER_URL = 'redis://localhost:6379/0'
+CELERY_RESULT_BACKEND = 'redis://localhost:6379/0'
+CELERY_CACHE_BACKEND = "django-chace"
+CELERY_ACCEPT_CONTENT = ['application/json']
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'json'
+CELERY_TIMEZONE = 'Europe/Berlin'
+CELERY_RESULT_EXPIRES = 3600
+
+try:
+    from resotool.local_settings import *
```

Comment:
One last nitpick, can we make this import qualified so as to know where the things in our namespace come from?

Reply:
In this case that's not really possible if we intent to use `local_settings.py` as I intended it.

A small example:
Once our code goes in production, we want to change `DEBUG` from `True` to `False` in `settings.py`. But in my local dev repo I still want to be able to set `DEBUG` to `True` without changing any tracked files in the repository. As it's set up now, the `DEBUG=True` setting from `local_settings.py` will override the `DEBUG=False` setting from `settings.py` and there is no way I could by accident push `DEBUG=True` setting to production.

If we changed this to a qualified import, we'd need a check for every setting, we want to allow to be overridden, whether there is a corresponding setting in `local_settings.py`. On the other hand, this would also give us some security in what can be overridden and what not.

Reply:
I see, but wouldn't a `git update-index --assume-unchanged settings.py` be another possibility to achieve the same thing as `local_settings.py`? This way there is no need to check in your development settings at all.

Reply:
The development settings shouldn't be checked in anyway. That's why I put `local_settings.py` into the `.gitignore`.

And yes, that would achieve the same thing. But in my experience it is easier to tell people to do it this way, than to teach them a new Git command which will also produce errors when `settings.py` changes upstream.

But it the end, I don't have a strong preference for one way over the other. If you prefer to use `git update-index --assume-unchanged settings.py`, I will remove the support for `local_settings.py`.

Reply:
Fair enough. The dance with `git update-index --assume-unchanged` can be tough if one doesn't know what one is doing. But we should finding a better solution on how to do this should go on our to do list. I don't quite favour this shadowing of configs via importing something. I think this is brittle, but for now it's good enough.